openshift-ansible.spec

```diff
@@ -79,103 +55,8 @@ cp -rp roles %{buildroot}%{_datadir}/ansible/%{name}/
-%package docs
-Summary:       Openshift and Atomic Enterprise Ansible documents
-Requires:      %{name} = %{version}-%{release}
-BuildArch:     noarch
-
-%description docs
-%{summary}.
-
-%files docs
 %doc  docs
-
-# ----------------------------------------------------------------------------------
-# openshift-ansible-playbooks subpackage
-# ----------------------------------------------------------------------------------
-%package playbooks
-Summary:       Openshift and Atomic Enterprise Ansible Playbooks
-Requires:      %{name} = %{version}-%{release}
-Requires:      %{name}-roles = %{version}-%{release}
-BuildArch:     noarch
-
-%description playbooks
-%{summary}.
-
-%files playbooks
 %{_datadir}/ansible/%{name}/playbooks
-
-# Along the history of openshift-ansible, some playbook directories had to be
-# moved and were replaced with symlinks for backwards compatibility.
-# RPM doesn't handle this so we have to do some pre-transaction magic.
-# See https://fedoraproject.org/wiki/Packaging:Directory_Replacement
-%pretrans playbooks -p <lua>
--- Define the paths to directories being replaced below.
--- DO NOT add a trailing slash at the end.
-dirs_to_sym = {
-    "/usr/share/ansible/openshift-ansible/playbooks/common/openshift-master/library",
-    "/usr/share/ansible/openshift-ansible/playbooks/certificate_expiry"
-}
-for i,path in ipairs(dirs_to_sym) do
-  st = posix.stat(path)
-  if st and st.type == "directory" then
-    status = os.rename(path, path .. ".rpmmoved")
-    if not status then
-      suffix = 0
-      while not status do
-        suffix = suffix + 1
-        status = os.rename(path .. ".rpmmoved", path .. ".rpmmoved." .. suffix)
-      end
-      os.rename(path, path .. ".rpmmoved")
-    end
-  end
-end
-
-%package roles
-# ----------------------------------------------------------------------------------
-# openshift-ansible-roles subpackage
-# ----------------------------------------------------------------------------------
-Summary:       Openshift and Atomic Enterprise Ansible roles
-Requires:      %{name} = %{version}-%{release}
-Obsoletes:      %{name}-lookup-plugins
-Obsoletes:      %{name}-filter-plugins
-Obsoletes:      %{name}-callback-plugins
-BuildArch:     noarch
-
-%description roles
-%{summary}.
-
-%files roles
 %{_datadir}/ansible/%{name}/roles
 
-%pretrans roles
-#RHBZ https://bugzilla.redhat.com/show_bug.cgi?id=1626048
-#roles/openshift_examples/latest used to be a symlink, now its a dir
-# workaround for RPM bug https://bugzilla.redhat.com/show_bug.cgi?id=975909
-if [ -d %{_datadir}/ansible/%{name}/roles/openshift_examples/files/examples ]; then
-  find %{_datadir}/ansible/%{name}/roles/openshift_examples/files/examples -name latest -type l -delete
-fi
-
-# ----------------------------------------------------------------------------------
-# openshift-ansible-tests subpackage
-# ----------------------------------------------------------------------------------
-%package test
```

Comment:
CI was updated to install this test package.  Should we do something different there?
https://github.com/openshift/openshift-ansible/blob/master/images/installer/Dockerfile#L30

Reply:
I'll add it back as a sub-package. I don't think we'd want to ship it in our channels though.

Reply:
I agree we should not ship openshift-ansible-test.